codecov.yml

```diff
@@ -10,3 +10,20 @@ coverage:
         # basic
         target: auto
         threshold: 3%
+
+comment:
+  layout: "header, diff, flags"
+  behavior: default
+  require_changes: false
+
+flag_management:
+  default_rules: # the rules that will be followed for any flag added, generally
+    carryforward: true
+    statuses:
+      - type: project
+        target: 70%
+      - type: patch
+        target: 70%
+
+ignore:
+  - tests/** # integration test cases or tools.
```

Comment:
Does ignoring here will reduce the test coverage?

Reply:
Maybe, but they are test tools or integration test cases, should be ignored.